Formula/c/cargo-clone.rb

```diff
@@ -0,0 +1,45 @@
+class CargoClone < Formula
+  desc "Cargo subcommand to fetch the source code of a Rust crate"
+  homepage "https://github.com/JanLikar/cargo-clone"
+  url "https://github.com/JanLikar/cargo-clone/archive/refs/tags/v1.2.4.tar.gz"
+  sha256 "58e86bd3440fc103572f6d8ff20ff1f99cf1e676cddb29975f204a9cb03f5b14"
+  license any_of: ["Apache-2.0", "MIT"]
+  head "https://github.com/JanLikar/cargo-clone.git", branch: "master"
+
+  depends_on "pkgconf" => :build
+  depends_on "rust" => :build
+  depends_on "rustup" => :test
+
+  depends_on "openssl@3"
```

Comment:
Actually, since this [seems to use the `curl` crate](https://github.com/JanLikar/cargo-clone/blob/54c890b478ef5758eb1b144a2a239d64993a315b/Cargo.lock#L652-L681), we should make sure this uses our `libcurl`.